.gitignore

```
@@ -1,5 +1,5 @@
 allure-results
 allure-report
-node_modules
+/node_modules
```

Why do we need this "/"?
```suggestion
node_modules
```

---

@ErikHP так у себя в проектах видел

---

@Burdishka that how we did in our projects) Check my page